docs/README.md

```diff
@@ -0,0 +1,9 @@
+# Mattermost/GitLab Integration
+
+[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-gitlab/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-gitlab) [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-gitlab/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-gitlab)
+
+This plugin supports a two-way integration between Mattermost and GitLab. For a stable production release, please download the latest version [in the Releases tab](https://github.com/mattermost/mattermost-plugin-gitlab/releases) and follow [these instructions](./#2-configuration) for install and configuration. Check the Plugin Marketplace.
```

Comment:
```suggestion
This plugin supports a two-way integration between Mattermost and GitLab. For a stable production release, please download the latest version [in the Releases tab](https://github.com/mattermost/mattermost-plugin-gitlab/releases) and follow [these instructions](./#2-configuration) for install and configuration. Check the Plugin Marketplace.
```

Is there a link to the Plugin Marketplace and are there next steps after this, or should we remove this sentence?

Reply:
This came from the JIRA plugin documentation.  This is the first page in the Gitbook (see https://mattermost.gitbook.io/plugin-jira/).  The next step is the next chapter in the Gitbook that is pointed to at the bottom of the Gitbook, which is Feature Summary, and then to Installation.  I think in the Gitbook whenever it refers to the Plugin Marketplace it is referring to the one you can get to by clicking on the hamburger menu on the top left when in mattermost .  However, if you think it is appropriate, I could put a link here to https://integrations.mattermost.com/.

Reply:
Ah that makes a lot of sense. @aaronrothschild, what would work best in this case? Instructions for accessing Plugin Marketplace via Mattermost, or a link to the integrations page?